### PR TITLE
Add RMIT University

### PR DESCRIPTION
**Pull request by xiaolul**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [x] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

File: csrankings.csv [hunk 1]
```diff
@@ -12995,3 +12995,36 @@ Zygmunt J. Haas,University of Texas at Dallas,http://www.utdallas.edu/~zjh130030
...
 Ümit V. Çatalyürek,Georgia Institute of Technology,http://bmi.osu.edu/umit/,OLDMURQAAAAJ
+Mark Sanderson,RMIT University,https://www.rmit.edu.au/contact/staff-contacts/academic-staff/s/sanderson-professor-mark,HmqvvQkAAAAJ
+Lawrence Cavedon,RMIT University,http://www1.rmit.edu.au/staff/lawrence-cavedon,ZAEiiEUAAAAJ
+J. Shane Culpepper,RMIT University,https://www.rmit.edu.au/contact/staff-contacts/academic-staff/c/culpepper-dr-shane,OFlBSEEAAAAJ
+Falk Scholer ,RMIT University,https://www.rmit.edu.au/contact/staff-contacts/academic-staff/s/scholer-associate-professor-falk,qOLjlqsAAAAJ
+James A. Thom,RMIT University,https://www.rmit.edu.au/contact/staff-contacts/academic-staff/t/thom-associate-professor-james,7ZXutEAAAAJ
+Xiuzhen Zhang,RMIT University,http://www1.rmit.edu.au/staff/xiuzhen-zhang,rIuehYoAAAAJ
+Zhifeng Bao,RMIT University,http://www1.rmit.edu.au/staff/zhifeng-bao,ck-yNM0AAAAJ
+Andy Song,RMIT University,http://www1.rmit.edu.au/staff/andy-song,sZ7dN0sAAAAJ
+Flora Dilys Salim,RMIT University,http://www1.rmit.edu.au/staff/flora-salim,Yz35RSYAAAAJ
```
Comment (emeryberger) [inline csrankings.csv]: Broken web link.

File: csrankings.csv [hunk 2]
```diff
@@ -12995,3 +12995,36 @@ Zygmunt J. Haas,University of Texas at Dallas,http://www.utdallas.edu/~zjh130030
 Özgür Simsek,University of Bath,https://www.mpib-berlin.mpg.de/en/staff/oezguer-simsek,z1BYZG0AAAAJ
 Öznur Özkasap,Koç University,http://home.ku.edu.tr/~oozkasap/,v2WiJeIAAAAJ
 Ümit V. Çatalyürek,Georgia Institute of Technology,http://bmi.osu.edu/umit/,OLDMURQAAAAJ
+Mark Sanderson,RMIT University,https://www.rmit.edu.au/contact/staff-contacts/academic-staff/s/sanderson-professor-mark,HmqvvQkAAAAJ
+Lawrence Cavedon,RMIT University,http://www1.rmit.edu.au/staff/lawrence-cavedon,ZAEiiEUAAAAJ
+J. Shane Culpepper,RMIT University,https://www.rmit.edu.au/contact/staff-contacts/academic-staff/c/culpepper-dr-shane,OFlBSEEAAAAJ
```
Comment (emeryberger) [inline csrankings.csv]: Home page should be http://www.culpepper.io/.

File: csrankings.csv [hunk 3]
```diff
@@ -12995,3 +12995,36 @@ Zygmunt J. Haas,University of Texas at Dallas,http://www.utdallas.edu/~zjh130030
 Özgür Simsek,University of Bath,https://www.mpib-berlin.mpg.de/en/staff/oezguer-simsek,z1BYZG0AAAAJ
 Öznur Özkasap,Koç University,http://home.ku.edu.tr/~oozkasap/,v2WiJeIAAAAJ
 Ümit V. Çatalyürek,Georgia Institute of Technology,http://bmi.osu.edu/umit/,OLDMURQAAAAJ
+Mark Sanderson,RMIT University,https://www.rmit.edu.au/contact/staff-contacts/academic-staff/s/sanderson-professor-mark,HmqvvQkAAAAJ
+Lawrence Cavedon,RMIT University,http://www1.rmit.edu.au/staff/lawrence-cavedon,ZAEiiEUAAAAJ
```
Comment (emeryberger) [inline csrankings.csv]: Dead link.

File: csrankings.csv [hunk 4]
```diff
@@ -12995,3 +12995,36 @@ Zygmunt J. Haas,University of Texas at Dallas,http://www.utdallas.edu/~zjh130030
...
+J. Shane Culpepper,RMIT University,https://www.rmit.edu.au/contact/staff-contacts/academic-staff/c/culpepper-dr-shane,OFlBSEEAAAAJ
+Falk Scholer ,RMIT University,https://www.rmit.edu.au/contact/staff-contacts/academic-staff/s/scholer-associate-professor-falk,qOLjlqsAAAAJ
+James A. Thom,RMIT University,https://www.rmit.edu.au/contact/staff-contacts/academic-staff/t/thom-associate-professor-james,7ZXutEAAAAJ
+Xiuzhen Zhang,RMIT University,http://www1.rmit.edu.au/staff/xiuzhen-zhang,rIuehYoAAAAJ
+Zhifeng Bao,RMIT University,http://www1.rmit.edu.au/staff/zhifeng-bao,ck-yNM0AAAAJ
+Andy Song,RMIT University,http://www1.rmit.edu.au/staff/andy-song,sZ7dN0sAAAAJ
+Flora Dilys Salim,RMIT University,http://www1.rmit.edu.au/staff/flora-salim,Yz35RSYAAAAJ
+Isaac Balbin,RMIT University,https://www.rmit.edu.au/contact/staff-contacts/academic-staff/b/balbin-associate-professor-isaac,YCMaqMYAAAAJ
+Ke Deng,RMIT University,http://www1.rmit.edu.au/staff/ke-deng,MuWCgskAAAAJ
```
Comment (emeryberger) [inline csrankings.csv]: It appears all the links starting with www1 are broken.

**Review by emeryberger — CHANGES_REQUESTED** [review]
Lots of broken links for home pages that need to be fixed.

File: csrankings.csv (hunk 4 above)
Comment (xiaolul) [reply csrankings.csv]: Fixed, can you please check that again? Thank you.

**Comment by xiaolul** [comment]
I have fixed the homepage issue.

File: csrankings.csv (hunk 1 above)
Comment (xiaolul) [reply csrankings.csv]: Fixed.

File: csrankings.csv (hunk 3 above)
Comment (xiaolul) [reply csrankings.csv]: changed

File: csrankings.csv (hunk 2 above)
Comment (xiaolul) [reply csrankings.csv]: changed

File: csrankings.csv (hunk 1 above)
Comment (emeryberger) [reply csrankings.csv]: These all need Google Scholar links or NOSCHOLARPAGE.

Please read the commit checklist.

**Pull request closed without merging**